<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>game_play</class>
 <widget class="QMainWindow" name="game_play">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>917</width>
    <height>645</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>917</width>
    <height>645</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>917</width>
    <height>645</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-image: url(:/new/prefix1/woman-with-sudoku-game-application-phone-at-the-park.jpg);</string>
   </property>
   <widget class="QPushButton" name="pb00_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>470</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb00_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solve">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>470</y>
      <width>99</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>CrossCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Bot Solver</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb00_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>470</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb00_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>420</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb00_9">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>420</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>510</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pb00_6">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>470</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb00_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>420</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>60</y>
      <width>381</width>
      <height>351</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>360</width>
      <height>270</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="iconSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>40</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>新建行</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>新建列</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="0" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="1" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="2" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="3" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="4" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="5" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="6" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="7" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="0">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="1">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="2">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="3">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="4">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="5">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="6">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="7">
     <property name="text">
      <string>0</string>
     </property>
    </item>
    <item row="8" column="8">
     <property name="text">
      <string>0</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pb00_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb00_3">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>520</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="close_btn_game">
     <property name="geometry">
      <rect>
       <x>870</x>
       <y>0</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>ForbiddenCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/closeee.svg);
background:rgba(0, 0, 80, 0.8);
color :rgba(255, 255, 255, 1);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>0</y>
       <width>51</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Handwriting</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>WaitCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>reset</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/sodoko-mainpage.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="next_game">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>570</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Viner Hand ITC</family>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>BusyCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>NEW GAME</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>360</y>
      <width>201</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Old English Text MT</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>player info</string>
    </property>
    <widget class="QLineEdit" name="player_name">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>16</number>
     </property>
     <property name="placeholderText">
      <string>amin</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="player_point">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>256</number>
     </property>
     <property name="placeholderText">
      <string>10</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>point :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>61</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MV Boli</family>
       <pointsize>8</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>userinfo:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_info">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="player_pass">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Monotype Corsiva</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="maxLength">
      <number>8</number>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>re#hf54b</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="styleSheet">
    <string notr="true">background:rgb(0, 0, 80, 0.8);
color :rgb(255, 255, 255);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
